README.md

```diff
@@ -52,12 +52,14 @@ Features:
 
 Limitations or not yet implemented:
 
+* This is a device stack, so the IO-controller / master / PLC side is not supported.
```

Comment:
May "so" shall be replaced with "this means"

Reply:
Done